tests/io/test_particle_collection.py

```diff
@@ -126,3 +126,13 @@ def test_find_subset():
         assert len(search_result) == 1
         assert gamma_from_subset.pid == 22
         assert gamma_from_subset is particle.DATABASE["gamma"]
+
+
+def test_pdg():
+    pdg_particle_collection = io.load_particle_collection()
+    # io.write(pdg_particle_collection, "pdg.yml")
+    assert len(pdg_particle_collection) == 541
+    d_meson = pdg_particle_collection["D0"]
```

Comment:
For the eventual #189, it would be better if we compare to the existing definitions in `particle_list.yml`. This would also be a more thorough test for `load_pdg`, for instance, it would have fished out the problem with (see comment elsewhere):
https://github.com/ComPWA/expertsystem/blob/cac9256a9985114c27ab7752e1c235798a5c3e7a/expertsystem/io/_pdg.py#L23

Note that this requires matching the particle names in `particle_list.yml` with those used in the `particle` package. That can be done through:
1. a `pytest.mark.parametrize` decorator, just like the 'mapping' used by @sebastianJaeger here:
 
 https://github.com/ComPWA/expertsystem/blob/29d82ec786c09028a3ef792146de43b731a15b4a/tests/state/test_particle.py#L44-L48

2. Renaming the entries of `particle_list.yml` accordingly. But this also requires modifying several tests and possibly the notebooks.

For sure it is worth the effort, because need to ensure that the quantum numbers of `particle_list.yml` are the same and that the `load_pdg` function works correctly.

Reply:
Working on a complete comparison, seems several particle definitions are not the same as what is imported from the PDG (c1ae4e32997bb62fed367c534195e21553ad7575)